### PR TITLE
Fix/css tweaks

### PR DESCRIPTION
**Pull request by Shawntru**
## Is this a fix or a feature?
Fix
## What does it fix?

- Changed defaults on search form to return more games if user searches by default params
- Remove comments
- Add search view display count for matches
- Broke down element styling to use classes in all scss files
- Update scss to use variables stored in _variables.scss
- Background now has gradient
- Update search display grid to be dynamic
- Move all top level element styling to App.scss
- Homepage now has additional styling including different sized diamonds
- Animations now play on search form click
- Restyle search display result cards
- Create loading animation for search display 
- Add interactive navbar styling and centering

## Where should the reviewer start?
From the top, navigate and ensure all changes are sufficient 
## Relevant Issues
#48

### CODE REVIEW

**Comment by brycemara** [comment]
Color me impressed. The design looks awesome, good work Shawn!

**Comment by Shawntru** [comment]
> Color me impressed. The design looks awesome, good work Shawn!

No, I've already done too much css coloring and I'm sick of it 😂

File: src/App/App.scss [hunk 1]
```diff
@@ -1,49 +1,31 @@
...
+  background: radial-gradient(
+    circle,
+    rgba(60, 107, 143, 1) 0%,
+    rgba(2, 46, 81, 1) 90%
+  );
   height: 100%;
+  width: 100%;
+}
+
+h3,
```
Comment (mdflynn) [inline src/App/App.scss:15]: Good cleanup here

File: src/SearchDisplay/SearchDisplay.scss [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+  font-size: 0.5em;
+  font-style: italic;
+}
+
+@keyframes loading {
+  33% {
+    transform: rotate(8deg);
+  }
+  66% {
+    transform: translateX(15px) translateX(15px) rotate(4deg);
```
Comment (mdflynn) [inline src/SearchDisplay/SearchDisplay.scss:33]: nice animation

**Review by mdflynn — APPROVED** [review]
Great styling to the site

**Pull request merged**